core/server/data/migrations/versions/4.0/16-remove-labs-setting.js

```diff
@@ -0,0 +1,23 @@
+const logging = require('../../../../../shared/logging');
+const {createIrreversibleMigration} = require('../../utils');
+
+module.exports = createIrreversibleMigration(async (knex) => {
+    let result = await knex('settings')
+        .where('key', '=', 'labs')
+        .select('value');
+
+    if (!result || !result[0]) {
+        logging.warn(`Could not find labs setting`);
+        result = [{}];
```

Comment:
I think you should `return` here, otherwise the migration will break on `result[0].value`.

Reply:
@tpatel I've decided the best thing here would be to completely remove the labs as a key altogether :sweat_smile: